WordPressUI/WordPressUI/FancyAlert/FancyButton.swift

```diff
@@ -0,0 +1,196 @@
+import Foundation
+import UIKit
+
+
+// MARK: - FancyButton
+//
+open class FancyButton: UIButton {
+
+    /// Style: Primary + Normal State
+    ///
+    @objc public dynamic var primaryNormalBackgroundColor = Primary.normalBackgroundColor
```

Comment:
Should we call `configureButton` if any of the appearance properties change?

Reply:
Good point!!. Let's do that, but instead, i'll split `configureButton` so we don't re-render the backgrounds each time an unrelated property is changed.